internal/server/v2/observation/golden/variable/result.json

```diff
@@ -6352,11 +6352,6 @@
         "wikidataId/Q506877": {}
       }
     },
-    "minimumWage": {
```

Comment:
is this deletion expected?

Reply:
wikidataId/Q1951 , minimumWage combination is missing in Production as well.